mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -1306,6 +1310,16 @@ notification_config {
 <a name="rotation_config"></a>The `rotation_config` block supports:
 
 * `enabled` (Optional) - Enable the roation in Secret Manager add-on for this cluster.
+* `rotation_interval` (Optional, Beta) - The interval between two consecutive rotations. Default rotation interval is 2 minutes.
```

Comment:
Beta here should be a markdown link to the beta provider description. See other examples in this file